Microservices/email-service/src/main/java/app/sportahub/emailservice/controller/EmailController.java

```diff
@@ -0,0 +1,23 @@
+package app.sportahub.emailservice.controller;
+
+import app.sportahub.emailservice.dto.request.EmailRequestDTO;
+import app.sportahub.emailservice.service.EmailService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/api/emails")
```

Comment:
Replace the `@RequestMapping("/api/emails")` by `@RequestMapping("/email")` and add this line to the server.properties file of the `email-service`
```yaml
server.servlet.context-path=${SERVER_SERVLET_CONTEXT_PATH:/api/email-service}
```